cryostatio/cryostat-web
# feat(toolbar): revamp datetime picker and date display

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #615 
Fixes #571 

## Description of the change:

- [x] Reduce toolbar size but merge date input and time input
- [x] Add support for selecting timezone for date filter.
- [x] Format `startTime` display column to be clearer, readable and open for localization support.
- [x] Use dayJs (only 2KB) for date utilities.
- [x] Rename category displays instead of using Object keys as category names.
- [x] Add support for choosing date format (locale) and timezone in Settings.
- [x] Fixed unit tests for new changes.

## Motivation for the change:

Reduce toolbar space and add support  for date search with selected timezone.

## How to manually test:
1. Go to Recordings tab.
2. Select any of date-related filters.
3. See new changes.
